service/version_bundles.go

```diff
@@ -109,11 +109,52 @@ func NewVersionBundles() []versionbundle.Bundle {
 				},
 			},
 			Dependencies: []versionbundle.Dependency{},
-			Deprecated:   false,
+			Deprecated:   true,
 			Name:         "aws-operator",
 			Time:         time.Date(2017, time.November, 29, 16, 16, 0, 0, time.UTC),
 			Version:      "0.2.0",
-			WIP:          true,
+			WIP:          false,
```

Comment:
I think this should be true. The cloud formation changes are still WIP.

Reply:
Ack. When it is WIP it must not be deprecated.